src/components/views/rooms/RoomHeader.js

```diff
@@ -225,12 +226,17 @@ module.exports = React.createClass({
                 roomName = this.props.room.name;
             }
 
+            const innerName =
+                <EmojiText element="div"
+                           className={ "mx_RoomHeader_nametext " + (settingsHint ? "mx_RoomHeader_settingsHint" : "") }
+                           title={ roomName }>{roomName}</EmojiText>;
```

Comment:
A few things here:
 - props should be `{"someValue"}` (no spaces)
 - children should be `{ someChild }`